modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/TranslateMVCRenderCommand.java

```diff
@@ -207,6 +208,22 @@ public String render(
 		);
 	}
 
+	private BooleanSupplier _getBooleanSupplier(ThemeDisplay themeDisplay) {
```

Comment:
Inline this method (keep the lambda as it was); change the TranslateDisplayContext to use an UnsafeSupplier, so that it can throw exceptions (you may need to change additional method signatures for that).

Reply:
thanks!